src/github.com/matrix-org/dendrite/roomserver/storage/sql.go

```diff
@@ -230,6 +237,11 @@ const insertEventStateKeyNIDSQL = "" +
 const selectEventStateKeyNIDSQL = "" +
 	"SELECT event_state_key_nid FROM event_state_keys WHERE event_state_key = $1"
 
+const selectEventStateKeyNIDsSQL = "" +
+	"SELECT event_state_key, event_state_key_nid FROM event_state_keys" +
+	" WHERE event_state_key = ANY($1)" +
```

Comment:
`ANY($1)` is like SQLite3's `IN` right? Is there a limit to the number of args you can give? If there is, is it possible for us to reach the limit?